### ISSUE TITLE
Releases from `v1.0` to `v1.1.3` contain outdatet CITATION.cff

### ISSUE THREAD

**Issue by schlauch** [DLR-SC/gitlab2prov#65]
- On the Zenodo page there are different versions but they all seem to point to version 0.5: https://zenodo.org/record/6792665
- However, the actual latest snapshort is version 1.1.3 (which is the current latest release)
- This makes it a bit hard when trying to cite the right version using the right citation meatdata.
- The problem might be that the CITATION.cff file still references version 0.5. It seems Zenodo is pulling out the citation metadata form that file. See also: https://twitter.com/ZENODO_ORG/status/1420357001490706442

**Comment by cdboer** [comment]
You're right, that is not how it is supposed to be. Zenodo does take the version number out of the citation metadata. I misunderstood the citation file and thought that the `.cff` file was 'attached' to our publication that cites the `v0.5` release. Don't ask how i thought that was supposed to work. Just read up on it and will fix the version number starting with the next release which should be scheduled for monday or tuesday.

As for the past zenodo releases that still reference the `v0.5` release: Do you know if we can change the referenced version number retroactively? Ideally i'd like all releases to have their correct citation file.

**Comment by schlauch** [comment]
I fear that you cannot change the deposited archive afterwards. Thus, I would change the Citation.cff to make sure it works for the next releases. In addition, I would change the metadata of each archived version on Zenodo manually. I.e., you can change the displayed metadata in Zenodo but not the uploaded files.

**Comment by cdboer** [comment]
I updated the citation file for the current release (`v1.1.4`). I'll keep the issue pinned until i fix the metadata of the archived versions.

**Comment by cdboer** [comment]
[This issue](https://github.com/citation-file-format/citation-file-format/issues/374) describes the behavior of citation files in conjunction with github and zenodo.
